### PR TITLE
Add no-op to the glossary

### LINKED ISSUE #2388: Add "no-op" to glossary
It's used in the docs and users may not know what it means.

### PR DESCRIPTION
**Pull request by kshitijarvikar**
## Added Hyperlinks to "no-op", for better understanding of no-op behavior to readers.
resolved #2388

### CODE REVIEW

**Comment by rzblue** [comment]
Instead of linking to an external document it should be added to the glossary (https://github.com/wpilibsuite/frc-docs/blob/main/source/docs/software/frc-glossary.rst) and linked inline like so: `:term:\`no-op\``

**Comment by kshitijarvikar** [comment]
look for updated PR, or are there any changes suggested.!

File: source/docs/software/frc-glossary.rst [hunk 1]
```diff
@@ -178,3 +178,6 @@ FRC Glossary
 
    transitory
       In :term:`NetworkTables`, a :term:`topic` that will disappear after the last :term:`publisher` stops publishing.
+
+   no-op
+      No op means no operation, which is a computer instruction that specifies no operation but takes up a small amount of storage space. When the computer processor encounters a no op instruction, it simply moves to the next sequential instruction. Read more about no-op on `Wikipedia <https://en.wikipedia.org/wiki/NOP_(code)>`_ .
```
Comment (sciencewhiz) [inline source/docs/software/frc-glossary.rst:183]: I think the part about taking up a small amount of storage space is irrelevant and confusing. I'd remove it

**Review by sciencewhiz — COMMENTED** [review]
As rzblue said, existing uses need to use the term syntax. I see at least two uses, one in the command scheduler article and one in the new for 2024 article

**Pull request closed without merging**